documentation/build.gradle

```diff
@@ -61,7 +65,7 @@ task publishDitaSource() {
         def ditaSourceZipLocation = "${buildDir}/libs/${ditaSourceZipName}"
         exec {
             commandLine 'curl', '--insecure', '-u', "${project.ext.artifactoryDeployerUsername}:${project.ext.artifactoryDeployerPassword}", '-X', 'PUT',
-                    "${project.ext.deployArtifactoryUrl}/${project.ext.artifactoryRepo}/com/synopsys/integration/${rootProject.name}/${version}/${ditaSourceZipName}", '-T', "${ditaSourceZipLocation}", '-f'
+                    "${project.ext.deployArtifactoryUrl}/${project.ext.artifactoryRepo}/com/blackduck/integration/${rootProject.name}/${version}/${ditaSourceZipName}", '-T', "${ditaSourceZipLocation}", '-f'
```

Comment:
@shantyk Rollback this to com/synopsys (original), we don't want old versions to get published on new bd artifacts location

Reply:
This change would only affect where the ditasource ZIP file is uploaded, is that correct? In a folder at our internal artifactory always, but either under /com/synopsys/integration or /com/blackduck/integration depending on if it is a 10.0.0+ version or not. Is my understanding correct?

Reply:
Correct